npm/vue/src/index.ts

```diff
@@ -357,14 +358,18 @@ export const mount = (
     // @ts-ignore
     const document: Document = cy.state('document')
 
-    document.body.innerHTML = ''
     let el = document.getElementById(ROOT_ID)
 
-    // If the target div doesn't exist, create it
-    if (!el) {
-      el = renderTestingPlatform(document.head.innerHTML)
+    // first make sure the previous Vue instance
+    // no longer exists
+    if (Cypress.vue) {
+      Cypress.vue.$destroy()
+      delete Cypress.vue
```

Comment:
Do we really need to run `delete`? Using delete keyword (especially for reused giant objects like Cypress) can lead to deoptimizations of v8. Maybe we can use `= undefined` instead? 

> some source https://stackoverflow.com/questions/59431589/javascript-performance-constraints-of-delete-keyword

Reply:
I agree. 
We don't even need to run the delete. 
I was thinking of avoiding to run `$destroy()` on an empty object but the overhead would be fairly small

The other good option would be if there was a `destroyed` flag on the vm instance, then we could condition on that.

I will delete the delete line for now.

Reply:
removed delete line